src/pretix/plugins/sendmail/views.py

```diff
@@ -29,6 +30,17 @@ class SenderView(EventPermissionRequiredMixin, FormView):
     def get_form_kwargs(self):
         kwargs = super().get_form_kwargs()
         kwargs['event'] = self.request.event
+        if 'from_log' in self.request.GET:
+            try:
+                from_log_id = self.request.GET.get('from_log')
+                message = LogEntry.objects.get(
+                    id=from_log_id,
+                    event=self.request.event,
+                    action_type='pretix.plugins.sendmail.sent'
+                ).display()
+            except LogEntry.DoesNotExist:
+                raise Http404(_('You supplied an invalid log entry ID'))
+        kwargs['initial'] = {'message': message}
```

Comment:
You need to but this into the ``try`` part, otherwise this fails if you try to send an email without using an old email as the base.